app.js

```diff
@@ -9,169 +9,200 @@ function testEvent(){
   var counterSeven = 0;
   var tallyScore = 0;
   var percentage = 0;
+  //
+  function questionOne() {
 
-  var userName = prompt('What\'s your name?');
-  alert('Hello, ' + userName + '. Thanks for coming to my site!');
-
-  var answerOne = prompt('Was I born in Newfoundland?');
-  answerOne = answerOne.charAt(0);
-  answerOne = answerOne.toUpperCase();
-
-
-  if (answerOne === myAnswerOne) {
-    alert('Very well done!');
-    tallyScore = tallyScore + 1;
-    console.log('Answer given was ' + answerOne);
-  } else {
-    alert('Wrong! Next question!');
-  };
-
-  console.log("Question one complete!");
-
-  var answerTwo = prompt('Did I go to university in Newfoundland?');
-  answerTwo = answerTwo.charAt(0);
-  answerTwo = answerTwo.toUpperCase();
-
-
-  if (answerTwo === myAnswerTwo) {
-    alert('Very well done!');
-    tallyScore = tallyScore + 1;
-  } else {
-    alert('Wrong! On to the next question!');
-  };
-
-  console.log("Question two complete!");
-
-  var answerThree = prompt('Do I speak French?');
-  answerThree = answerThree.charAt(0);
-  answerThree = answerThree.toUpperCase();
-
-
-  if (answerThree === myAnswerThree) {
-    alert('Very well done!');
-    tallyScore = tallyScore + 1;
-  } else {
-    alert('Wrong! On to the next question!');
-  };
-
-  console.log("Question three complete");
-
-  var answerFour = prompt('Have I lived in the Yukon Territory?');
-  answerFour = answerFour.charAt(0);
-  answerFour = answerFour.toUpperCase();
-
-
-  if (answerFour === myAnswerFour) {
-    alert('Very well done!');
-    tallyScore = tallyScore + 1;
-  } else {
-    alert('Wrong! On to the next question!');
-  };
+    var userName = prompt('What\'s your name?');
+    alert('Hello, ' + userName + '. Thanks for coming to my site!');
+    window.userName = userName;
```

Comment:
Interesting that you're storing the userName variable as an object property onto the window object. Any reason for doing this?